python/py-apache-beam/Portfile

```diff
@@ -0,0 +1,90 @@
+maintainers         nomaintainer
+
+description         Apache Beam SDK for Python
+
+long_description    Apache Beam is a unified model for defining both\
+                    batch and streaming data-parallel processing\
+                    pipelines, as well as a set of language-specific\
+                    SDKs for constructing pipelines and Runners for\
+                    executing them on distributed processing backends,\
+                    including Apache Apex, Apache Flink, Apache Spark,\
+                    Google Cloud Dataflow and Hazelcast Jet.
+
+homepage            https://beam.apache.org
+master_sites        pypi:a/apache-beam
+use_zip             yes
+extract.mkdir       no
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  dc243db6c9fffab8cc82f1d62822fb3eae5b983b \
+                    sha256  1843d0cfc9f945e90d21e3d94b59e82613d0d5cbc44ae7befa65872f114e6b94 \
+                    size    1621014
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_run-append \
+                    port:py${python.version}-arrow \
+                    port:py${python.version}-avro \
+                    port:py${python.version}-crcmod \
+                    port:py${python.version}-dateutil \
+                    port:py${python.version}-dill \
+                    port:py${python.version}-fastavro \
+                    port:py${python.version}-grpcio \
+                    port:py${python.version}-grpcio-tools \
+                    port:py${python.version}-hdfs \
+                    port:py${python.version}-httplib2 \
+                    port:py${python.version}-mock \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-pip \
+                    port:py${python.version}-pymongo \
+                    port:py${python.version}-oauth2client \
+                    port:py${python.version}-protobuf3 \
+                    port:py${python.version}-pydot \
+                    port:py${python.version}-tz
+
+    depends_test-append \
+                    port:py${python.version}-freezegun \
+                    port:py${python.version}-nose \
+                    port:py${python.version}-nose_xunitmp \
+                    port:py${python.version}-pandas \
+                    port:py${python.version}-parameterized \
+                    port:py${python.version}-pyhamcrest \
+                    port:py${python.version}-pylint \
+                    port:py${python.version}-pytest \
+                    port:py${python.version}-pytest-xdist \
+                    port:py${python.version}-requests-mock \
+                    port:py${python.version}-tenacity \
+                    port:py${python.version}-yaml
+
+    if { ${python.version} == 27 } {
+        depends_run-append \
+                    port:py${python.version}-typing
+    }
```

Comment:
This block is no longer necessary and should be removed.

Reply:
Done thanks.